### PR TITLE
bpo-29607: Fix stack_effect computation for CALL_FUNCTION_EX

### PR DESCRIPTION
**Pull request by MatthieuDartiailh**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow these steps to rectify the issue:

1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
2. **Wait at least one US business day** and then check "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
3. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

**Review by methane — APPROVED** [review]

**Review by methane — CHANGES_REQUESTED** [review]
Could you add an entry to `Misc/NEWS` ?

It might be like this:

```
Core and Builtins
-----------------

- bpo-29607: Fix stack effect computation for CALL_FUNCTION_EX opcode.
```

**Review by methane — APPROVED** [review]

File: Misc/NEWS [hunk 1]
```diff
@@ -10,6 +10,8 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-29607: Fix stack_effect computation for CALL_FUNCTION_EX
```
Comment (serhiy-storchaka) [inline Misc/NEWS]: Please add "Patch by `<yourname>`."

File: Misc/NEWS [hunk 2]
```diff
@@ -10,6 +10,9 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-29607: Fix stack_effect computation for CALL_FUNCTION_EX. 
+  Patch by Matthieu Dartiailh
```
Comment (serhiy-storchaka) [inline Misc/NEWS]: Add a period at the end of the entry.

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by methane** [comment]
@MatthieuDartiailh I'm sorry, but I missed "CLA not signed" label.
Could you sign CLA?  Or have you already signed and waiting it's reviewed?

**Comment by MatthieuDartiailh** [comment]
It is waiting to be reviewed.

**Comment by serhiy-storchaka** [comment]
I think signing CLA is not required for such trivial change. Since the bug was reported anyone could write the patch.

But if Matthieu wants to continue his contributing, signing CLA is required.

**Pull request merged**